pkg/oauth/apis/oauth/v1/types.go

```diff
@@ -37,6 +37,9 @@ type OAuthAccessToken struct {
 
 	// RefreshToken is the value by which this token can be renewed. Can be blank.
 	RefreshToken string `json:"refreshToken,omitempty" protobuf:"bytes,9,opt,name=refreshToken"`
+
+	// TimeoutsIn is the seconds from CreationTime before this token timeouts.
+	TimeoutsIn int32 `json:"timeoutsIn,omitempty" protobuf:"varint,10,opt,name=timeoutsIn"`
```

Comment:
`*int32`

nil = use default
0 = no timeout

Will need to confirm that protobuf can tell the difference.

Reply:
The default is 0, ie no timeout (and will always be), I do not see any reason to distinguish here or elsewhere